MainMenu/src/mainmenu/view/Main.java

```diff
@@ -6,11 +6,11 @@
 import javafx.scene.Scene;
 import javafx.stage.Stage;
 
-public class UserInterface extends Application {
+public class Main extends Application {
 
   @Override
   public void start(Stage primaryStage) throws Exception {
-    Parent root = FXMLLoader.load(getClass().getResource("mainmenu.fxml"));
+    Parent root = FXMLLoader.load(getClass().getResource("MainMenuView.fxml"));
```

Comment:
Any reason this is changed? The convention everyone else is using is lowercase module name.

Reply:
This is an FXML file. The naming convention I thought we agreed on was the same as a Class file.